[2024-10-10 13:38:52] [INFO    ] discord: Created Discord Bot Instance.
[2024-10-10 13:38:52] [INFO    ] discord.client: logging in using static token
[2024-10-10 13:38:53] [INFO    ] discord: WAITING UNTIL READY: Update Status
[2024-10-10 13:38:53] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 0a148494ef23b502520b5e818f97d084).
[2024-10-10 13:38:55] [INFO    ] discord: RAN TASK: Update Status
[2024-10-10 13:39:55] [INFO    ] discord: RAN TASK: Update Status
[2024-10-10 13:40:55] [INFO    ] discord: RAN TASK: Update Status
[2024-10-10 13:41:55] [INFO    ] discord: RAN TASK: Update Status
[2024-10-10 13:42:22] [INFO    ] discord: Created Discord Bot Instance.
[2024-10-10 13:42:22] [INFO    ] discord.client: logging in using static token
[2024-10-10 13:42:22] [INFO    ] discord: WAITING UNTIL READY: Update Status
[2024-10-10 13:42:22] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 78c0c4df8c23b1b8dc1e148bb94f0d69).
[2024-10-10 13:42:24] [INFO    ] discord: RAN TASK: Update Status
[2024-10-10 13:43:03] [ERROR   ] discord.app_commands.tree: Ignoring exception in command 'send-message'
Traceback (most recent call last):
  File "c:\Users\osterling\Documents\GitHub\Discord.py-Bot-TEMPLATE\cogs\message.py", line 35, in send_message
    response = await interaction.response.send_message(embed=emb_confrm, ephemeral=True)
                                                             ^^^^^^^^^^
NameError: name 'emb_confrm' is not defined. Did you mean: 'emb_confirm'?

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\osterling\AppData\Roaming\Python\Python312\site-packages\discord\app_commands\commands.py", line 857, in _do_call
    return await self._callback(self.binding, interaction, **params)  # type: ignore
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\osterling\Documents\GitHub\Discord.py-Bot-TEMPLATE\cogs\message.py", line 42, in send_message
    if response:
       ^^^^^^^^
UnboundLocalError: cannot access local variable 'response' where it is not associated with a value

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\osterling\AppData\Roaming\Python\Python312\site-packages\discord\app_commands\tree.py", line 1310, in _call
    await command._invoke_with_namespace(interaction, namespace)
  File "C:\Users\osterling\AppData\Roaming\Python\Python312\site-packages\discord\app_commands\commands.py", line 883, in _invoke_with_namespace
    return await self._do_call(interaction, transformed_values)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\osterling\AppData\Roaming\Python\Python312\site-packages\discord\app_commands\commands.py", line 876, in _do_call
    raise CommandInvokeError(self, e) from e
discord.app_commands.errors.CommandInvokeError: Command 'send-message' raised an exception: UnboundLocalError: cannot access local variable 'response' where it is not associated with a value
[2024-10-10 13:43:24] [INFO    ] discord: RAN TASK: Update Status
[2024-10-10 13:44:24] [INFO    ] discord: RAN TASK: Update Status
[2024-10-10 13:45:24] [INFO    ] discord: RAN TASK: Update Status
[2024-10-10 13:46:24] [INFO    ] discord: RAN TASK: Update Status
[2024-10-10 13:47:24] [INFO    ] discord: RAN TASK: Update Status
[2024-10-10 13:48:24] [INFO    ] discord: RAN TASK: Update Status
[2024-10-10 13:49:24] [INFO    ] discord: RAN TASK: Update Status
[2024-10-10 13:50:24] [INFO    ] discord: RAN TASK: Update Status
[2024-10-10 13:51:24] [INFO    ] discord: RAN TASK: Update Status
[2024-10-10 13:52:24] [INFO    ] discord: RAN TASK: Update Status
[2024-10-10 13:53:24] [INFO    ] discord: RAN TASK: Update Status
[2024-10-10 13:54:24] [INFO    ] discord: RAN TASK: Update Status
[2024-10-10 13:55:24] [INFO    ] discord: RAN TASK: Update Status
[2024-10-10 13:56:24] [INFO    ] discord: RAN TASK: Update Status
[2024-10-10 13:57:24] [INFO    ] discord: RAN TASK: Update Status
[2024-10-10 13:58:24] [INFO    ] discord: RAN TASK: Update Status
[2024-10-10 13:59:24] [INFO    ] discord: RAN TASK: Update Status
[2024-10-10 14:00:24] [INFO    ] discord: RAN TASK: Update Status
[2024-10-10 14:01:24] [INFO    ] discord: RAN TASK: Update Status
[2024-10-10 14:02:24] [INFO    ] discord: RAN TASK: Update Status
